packages/libwnck.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Libwnck < Package
+class Libwnck < Meson
   description 'Library for layout and rendering of text'
   homepage 'http://www.gnome.org/'
   version '3.32.0'
```

Comment:
Update to [43.0](https://download.gnome.org/sources/libwnck/43/libwnck-43.0.tar.xz)